src/Jackett.Common/Indexers/TorrentDay.cs

```diff
@@ -90,6 +90,7 @@ public TorrentDay(IIndexerConfigurationService configService, WebClient wc, Logg
             AddCategoryMapping(1, TorznabCatType.MoviesSD, "Movies/XviD");
 
             AddCategoryMapping(17, TorznabCatType.Audio, "Music/Audio");
+            AddCategoryMapping(17, TorznabCatType.AudioMP3, "Music/Audio");
```

Comment:
no point adding duplicate categories, as only one will take precedent (i forget if its lifo or fifo)
so please remove the old cat17

Reply:
done!